include/OpenSpaceToolkit/Physics/Environment/Gravitational/Earth.hpp

```diff
@@ -112,6 +119,10 @@ class Earth : public Model
 
     virtual Vector3d getFieldValueAt(const Vector3d& aPosition, const Instant& anInstant) const override;
 
+    static Model::Parameters ParametersFromType(const Earth::Type& aType);
+
+    static constexpr double gravityConstant = 9.80665;  /// Standard gravity [m.s-2]
```

Comment:
Isn't this defined in a constants file somewhere else already?

Reply:
Not that I could find, I can move this to ostk::physics::environment::constants if appropriate there. I figured this was another good place for it.